info/firmware.info.xml

```diff
@@ -25,7 +25,7 @@
     <board type="amc">
         <firmware>
             <file>../ETH/AMC/amc.hex</file>
-            <version major='2' minor='11'/>
+            <version major='2' minor='12'/>
```

Comment:
Hi @ale-gi, I see that you have written v 2.12. That is correct because I recently merged the v 2.11 version. 

So you must change the title of the PR in `AMC v2.12`.

Then, just a question: you produced the binary using `robotology/devel` after the PR was merged, right? If so it is OK.

Reply:
> 
> Then, just a question:
> 
> * you produced the binary using `robotology/devel` after the PR was merged, right?
> 
> If so it is OK.

Hi @marcoaccame, yes, they binary has been produced after the merge, so that they are built from the latest devel.